pkg/metrics/metrics.go

```diff
@@ -11,19 +11,19 @@ import (
 
```

Comment:
can you please reword this commit to the specific corrections is doing and follow imperative style https://chris.beams.io/posts/git-commit/?